src/test/java/seedu/address/testutil/TestUtil.java

```diff
@@ -52,4 +54,11 @@ public static Index getLastIndex(Model model) {
     public static CalendarEvent getPerson(Model model, Index index) {
         return model.getFilteredCalendarEventList().get(index.getZeroBased());
     }
+
+    /**
+     * Returns the todolistevent in the {@code modelToDo}'s todolistevent list at {@code index}.
+     */
+    public static ToDoListEvent getTask(ModelToDo modelToDo, Index index) {
```

Comment:
will changing this affect the tests related to the calendar event list?

Reply:
nope They can all pass
only addToDoSystemTest and deleteToDoSystemTest cannot pass